doc/source/whatsnew/v0.15.1.txt

```diff
@@ -110,19 +110,22 @@ API changes
 
   .. code-block:: python
 
-    In [8]: s.loc[3.5:1.5]
-    KeyError: 3.5
+     In [8]: s.loc[3.5:1.5]
+     KeyError: 3.5
 
   current behavior:
 
-  .. ipython:: python
-
-    s.loc[3.5:1.5]
+  .. code-block:: python
 
+     In [2]: s.loc[3.5:1.5]
+     Out[2]:
+     3    b
+     2    c
+     dtype: object
```

Comment:
This should not be needed (when #12370 is merged) I think? So would leave this out as well